utils/config/config.js

```diff
@@ -161,11 +164,10 @@ function populateFromStorage(org, orgList, site, siteList) {
       const { orgs } = projects;
       populateList(orgList, orgs);
       // populate org field
-      const lastOrg = projects.orgs[0];
-      setFieldValue(org, lastOrg, 'storage');
-      if (projects.sites && projects.sites[lastOrg]) {
+      const selectedOrg = setFieldValue(org, projects.orgs[0], 'storage');
```

Comment:
if an org was selected in previous population step, use that org. prevents adding sites to data list that are from a different org.